docs/site/package.json

```diff
@@ -35,6 +35,7 @@
   },
   "dependencies": {
     "marked": "^0.3.6",
+    "material-ui-icons": "^1.0.0-alpha.2",
```

Comment:
I think this is simplest for now. We can always revisit it if it causes unanticipated issues.